<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GTAActionIRTWidget</class>
 <widget class="QWidget" name="GTAActionIRTWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resource.qrc">
    <normaloff>:/images/forms/img/GTAAppIcon.png</normaloff>:/images/forms/img/GTAAppIcon.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="FunctiionLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>IRT Sequence</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="SequenceFileLE"/>
     </item>
     <item>
      <widget class="QPushButton" name="SearchPB">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/images/forms/img/SEARCH.png</normaloff>:/images/forms/img/SEARCH.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sequence</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QListWidget" name="SequenceListLW"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="PreviewLabel">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QTextEdit" name="PreviewTE">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>SequenceFileLE</tabstop>
  <tabstop>SearchPB</tabstop>
  <tabstop>SequenceListLW</tabstop>
  <tabstop>PreviewTE</tabstop>
 </tabstops>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
